src/App/components/Header.js

```diff
@@ -0,0 +1,69 @@
+import React, { useContext, useState } from 'react';
+import styled from 'styled-components';
+import { TaskContext } from '../contexts/taskContext';
+
+const BaseInput = styled.input`
+  position: relative;
+  margin: 0;
+  width: 100%;
+  font-size: 24px;
+  font-family: inherit;
+  font-weight: inherit;
+  line-height: 1.4em;
+  border: 0;
+  color: inherit;
+  padding: 6px;
+  border: 1px solid #999;
+  box-shadow: inset 0 -1px 5px 0 rgba(0, 0, 0, 0.2);
+  box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+`;
+
+const AddTodo = styled(BaseInput)`
+  padding: 16px 16px 16px 60px;
+  border: none;
+  background: rgba(0, 0, 0, 0.003);
+  box-shadow: inset 0 -2px 1px rgba(0,0,0,0.03);
+`;
+
+const Title = styled.h1`
+  position: absolute;
+  top: -155px;
+  width: 100%;
+  font-size: 100px;
+  font-weight: 100;
+  text-align: center;
+  color: rgba(175, 47, 47, 0.15);
+  -webkit-text-rendering: optimizeLegibility;
+  -moz-text-rendering: optimizeLegibility;
+  text-rendering: optimizeLegibility;
+`;
+
+const Header = () => {
+  const [input, setInput] = useState('');
+  const { postData } = useContext(TaskContext);
+  const onEnter = (e) => {
+    if (e.key === 'Enter' && input) {
```

Comment:
Considering using constant for KEY_VALUES

Reply:
I fixed this